coreprofile/10/_index.md

```diff
@@ -0,0 +1,36 @@
+---
+title: "Jakarta EE Core Profile 10 (under development)"
+date: 2021-04-07
+summary: "Release of the Jakarta EE Core Profile 10"
+---
+The Jakarta EE Core Profile defines a profile of the Jakarta EE platform specifically targeted at smaller runtimes.
+
+The goals for this release are:
+1. To provide a profile that contains a set of Jakarta EE Specifications targeting smaller runtimes suitable for microservices as well as enabling compilation to native images.
```

Comment:
At this point I would object to inclusion of native images as a requirement of the Core Profile.

Reply:
I agree with Steve.

Reply:
There would be no requirements for native images. The changes being suggested are to enable such platforms to have a profile they can be based on.

Reply:
Also, native images are not the main point.  Being able to create the application image at build time vs runtime is the focus of the CDI-lite changes and the core profile support.

Reply:
I agree with Steve. We need to remove `native-image` part. The scope statement must be minimal to allow movements. Changing it is complicated. `native-image` support may break compatibility with other profiles. This topic must be discussed in the core-profile group after the project is created.

Reply:
Change  ".. as enabling ..." to "as enabling build time oriented implementations."

Reply:
Would it be more acceptable to use the word "allowing" instead of "enabling"?  We don't want to preclude the creation of native images or build-time oriented impls.  But, at the same time, we're not looking to do anything special to enable this creation.  So, maybe we should just indicate that the Core Profile would allow for this type of impl creation?

Reply:
Wording was adjusted based on discussions during the Platform call.